i/is-weakmap/is-weakmap.sh

```diff
@@ -0,0 +1,74 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: is-weakmap
+# Version	: v2.0.1
+# Source repo	: https://github.com/inspect-js/is-weakmap
+# Tested on	: UBI 8.5
+# Language      : Ruby
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=is-weakmap
+PACKAGE_VERSION=v2.0.1
```

Comment:
Make this configurable from command line.  
```suggestion
PACKAGE_VERSION=${1:-v2.0.1}
```